var/spack/repos/builtin/packages/heppdt/package.py

```diff
@@ -0,0 +1,19 @@
+from spack import *
+
+class Heppdt(Package):
+    """ HEPPDT ."""
```

Comment:
Why not a more informative docstring, like:

> The HepPID library contains translation methods for particle ID's
to and from various Monte Carlo generators and the PDG standard
numbering scheme.  We realize that the generators adhere closely
to the standard, but there are occasional differences.

This comes from the package's README